openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml

```diff
@@ -0,0 +1,32 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  annotations:
+    description: Build a container and push it to image streams
+  labels:
+    app: wordpress-tungtest
```

Comment:
is this as intended?

Reply:
There are different ways to build and push the container image, this is one of the ways.
Or if the question is about WP 5.0.2, we need to patch WP core for different sites so yes we need to have one

Reply:
I just wonder about the label, it sounds like a personal test on local environment

Reply:
Not really a test, the namespace can be changed.
And, if we do not have the one built for openshift namespace, this is the one done for a specific environment/project but taking the same s2i source. We have to patch the sites anyway.